pkgs/development/python-modules/git-find-repos/default.nix

```
@@ -0,0 +1,27 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+  setuptools-scm,
+}:
+buildPythonPackage rec {
+  pname = "git-find-repos";
+  version = "2.1.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "acroz";
+    repo = "git-find-repos";
+    rev = version;
+    sha256 = "sha256-4TuZlt6XH4//DBHPuIMl/i3Tp6Uft62dGCTAuZ2rseE=";
+  };
+
+  build-system = [setuptools-scm];
```

```suggestion
  build-system = [ setuptools-scm ];
```

It's better to format with `nixfmt-rfc-style` that compliants with RFC 0166 (ref: https://github.com/NixOS/rfcs/pull/166).

---

Thanks! Honestly I'd expect some out-of-the-box pre-commit or treefmt integration at this point if nixpkgs cares about formatting.

Done manually nevertheless.